test/integrations/destinations/af/router/data.js

```diff
@@ -0,0 +1,398 @@
+        "description": "Test 1",
+        "feature": "router",
+        "module": "destination",
+        "version": "v0",
+        "input": {
+          "request": {
+            "body":{"input":[{
+                "message": {
+                  "channel": "web",
+                  "context": {
+                    "externalId": [
+                      {
+                        "type": "appsflyerExternalId",
+                        "id": "afUid"
+                      }
+                    ],
+                    "app": {
+                      "build": "1.0.0",
+                      "name": "RudderLabs JavaScript SDK",
+                      "namespace": "com.rudderlabs.javascript",
+                      "version": "1.0.0"
+                    },
+                    "traits": {
+                      "email": "testhubspot2@email.com",
+                      "name": "Test Hubspot",
+                      "anonymousId": "12345"
+                    },
+                    "library": {
+                      "name": "RudderLabs JavaScript SDK",
+                      "version": "1.0.0"
+                    },
+                    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36",
+                    "locale": "en-GB",
+                    "ip": "0.0.0.0",
+                    "os": {
+                      "name": "android",
+                      "version": ""
+                    },
+                    "screen": {
+                      "density": 2
+                    }
+                  },
+                  "type": "page",
+                  "messageId": "e8585d9a-7137-4223-b295-68ab1b17dad7",
+                  "originalTimestamp": "2019-10-15T09:35:31.289Z",
+                  "anonymousId": "00000000000000000000000000",
+                  "userId": "12345",
+                  "properties": {
+                    "path": "",
+                    "referrer": "",
+                    "search": "",
+                    "title": "",
+                    "url": ""
+                  },
+                  "name": "ApplicationLoaded",
+                  "sentAt": "2019-10-14T11:15:53.296Z",
+                  "integrations": {
+                    "AF": {
+                      "af_uid": "afUid"
+                    }
+                  }
+                },
+                "metadata": {
+                  "jobId": 2
+                },
+                "destination": {
+                  "Config": {
+                    "devKey": "ef1d42390426e3f7c90ac78272e74344",
+                    "sharingFilter": "hey",
+                    "androidAppId": "com.rudderlabs.javascript"
+                  },
+                  "Enabled": true
+                }
+              }],"destType": "af"},
+            "method": "POST"
+          },
+          "pathSuffix": ""
+        },
+        "output": {
+          "response": {
+            "status": 200,
```

Comment:
Are there any negative test cases for this destination?
For example, an input that should return 4xx.